Issue#930 -PoC-notes.md

```diff
@@ -0,0 +1,27 @@
+# Issue#930 PoC, Support Multiple Mappings Accessed Through Routes

```

Comment:
Can you migrate this Multiple Mapping Definition access usage into docs/src/main/asciidoc/user-guide/?

Reply:
Sure!

Reply:
@igarashitm This reminds me with a question. I intended not to commit changes in generated swagger API docs, but the thing making me unsure is that they are not under `.gitignore` . Shall i commit them?

Reply:
Oh yes please, they're auto-generated but still to be committed. As you changed REST API it would change the swagger doc as well.

Reply:
So now it's safe to remove this file?

Reply:
Yes it is. Actually i thought i did. I will delete it

Reply:
done